src/brandguard/tasks.py

```
@@ -0,0 +1,123 @@
+                callback_url,
+                json=payload,
+                headers={"X-Internal-Secret": _INTERNAL_SECRET},
+                timeout=30,
+            )
+            logger.info(
+                "[%s] Callback sent → %s (HTTP %s)",
+                job_id,
+                callback_url,
+                resp.status_code,
+            )
+        except Exception as cb_err:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)